### ISSUE TITLE
can't set notification time beyond midnight?

### ISSUE THREAD

**Issue by matthiaskrgr** [stom79/mastalab#370]
Mastalab 1.8.4
instance: chaos.social

Hi,  I'm trying to set the notification time to "from 05:00 to 01:00" so that from 1 to 5 it is quiet and from 5 to 1 I would get notifications, but it seems I cannot set the second time beyond the 23:00 limit "Time must be greater than 05:00".

Is there a way to work around that?

**Comment by stom79** [comment]
Yes. My bad. I didn't think it could be set in this direction. I will fix that so that this configuration works.

**Comment by matthiaskrgr** [comment]
Awesome, thanks! :D

**Comment by stom79** [comment]
Currently, you can set 00:00 and 05:00, you will lose one hour. But I will fix that asap.
